tasks/push-disk-images/README.md

```diff
@@ -1,6 +1,7 @@
 # push-disk-images
 
 Tekton task to push disk images via an InternalRequest. The environment to use is pulled from the `cdn.env` key in the data file.
+This task additionally pushes the disk images to Developer Portal
```

Comment:
It reads weird now: It pushes disk images via IR... and it also pushes the images to developer portal.

Where was it pushing it in the first place? Exodus CDN? Maybe add that bit to the first sentence?

But also, it seems it's still all happening via that same internal request.

Also, missing period at the end of sentence.

So maybe something like:
```
Tekton task to push disk images via an InternalRequest to Exodus CDN and also optionally to Developer Portal. The environment to use is pulled from the `cdn.env` key in the data file.
```
Or if you can come up with something better...

Reply:
updated. the A/C changed such that we push to both locations now...it is no longer optional.